File: search/linear_search.py
l = int(input("Enter number of elements in the list:"))
li = []
print("Enter elements one by one:")
for x in range (0,l):
	li.append(int(input()))
etf = int(input("Enter element to find: "))
found = False
location = -1
counter = 0
for x in li:
	if(x==etf):
		found = True
		location = counter
		break
	counter+=1
if(found):
	print("Element "+str(etf)+" found at "+str(location))
else:
	print("Element not found")

File: search/linear_search.java
import java.util.Scanner;
class LinearSearch
{
   public static void main(String args[])
   {
      int counter, numOfElements, itemToSearch, array[];
      Scanner scanner = new Scanner(System.in);
      System.out.println("Enter number of elements: ");
      numOfElements = scanner.nextInt(); 
      array = new int[numOfElements]; 
      System.out.println("Enter elements of array: ");
      for (counter = 0; counter < numOfElements; counter++)
        array[counter] = scanner.nextInt();
      System.out.println("Enter element to search: ");
      itemToSearch = scanner.nextInt();
      for (counter = 0; counter < numOfElements; counter++)
      {
         if (array[counter] == itemToSearch) 
         {
           System.out.println(itemToSearch+" is present at "+(counter+1));
           break;
         }
      }
      if (counter == numOfElements)
        System.out.println(itemToSearch + " is not present.");
   }
}